### PR TITLE
fixed pending test closes #123

### LINKED ISSUE #123: fix skipped spec 'the admin view home page shows an admin link on the homepage when admin logged it'

### PR DESCRIPTION
**Pull request by mikedao**
Unskipped our one pending test.  Changed the layout to have the link go where it's supposed to go. Modified the test file in question to use Factory Girl.

### CODE REVIEW

File: spec/integration/admin_view_spec.rb [hunk 1]
```diff
@@ -4,14 +4,16 @@
   include Capybara::DSL
 
   before(:each) do
```
Comment (nathanows) [inline spec/integration/admin_view_spec.rb:6]: before each, mike? and then going to a new route in the first test? (yeah, i know i wrote this test... :8ball: ). should we strip the before out before pushing? (ben said make comments in question form right? :snowboarder: :last_quarter_moon:

Comment (mikedao) [inline spec/integration/admin_view_spec.rb:6]: You can't give me crap about code you wrote!  ;-p  I'll fix it in the morning.

Comment (nathanows) [inline spec/integration/admin_view_spec.rb:6]: haha, i know. i'll go in and fix it now they were my shitty tests. will just need to have emily or krista merge it tomorrow.

File: app/views/shared/_nav_bar.html.erb [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <%= menu_item "Home", root_path %>
     <%= menu_item "Menu", menu_path %>
     <% if current_user && current_user.admin? %>
-      <%= menu_item "Admin", "#" %>
+      <%= menu_item "Admin", admin_path %>
```
Comment (EmilyMB) [inline app/views/shared/_nav_bar.html.erb:6]: Nice.

**Comment by EmilyMB** [comment]
Why is it saying you changed all the images?

**Comment by EmilyMB** [comment]
There are a lot of files that seem extraneous such as: public/assets/twitter/bootstrap/glyphicons-halflings-932d74bb919875de8ad480f4279100f2.png

**Comment by mikedao** [comment]
I think the extra files are a git thing because I merged in nathans changes and then pushed

**Pull request merged**